profiles/arch/riscv/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 2019-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# ArchFeh <guyu2876@gmail.com> (2022-07-26)
```

Comment:
I think it's actually better to have your real name here and to match the sign-off, I'll fix it while mergine

Reply:
OK